debian/rules

```diff
@@ -67,7 +67,7 @@ clean:
 	  ltmain.sh missing
 	rm -f configure Makefile.in man/Makefile.in src/Makefile.in
 	rm -f src/acconfig.h.in
-	rm -f debian/ceph-base.init debian/radosgw.init debian/ceph-base/etc/logrotate.d/ceph.logrotate debian/radosgw.logrotate
+	rm -f debian/ceph-base.init debian/radosgw.init debian/ceph.logrotate debian/radosgw.logrotate
```

Comment:
this change and the change above it undo the changes I made before merged, does ceph.logrotate not belong in debian/ceph-base/etc/logrotate.d/ anymore? Or did the changes I made not accomplish that? I remember it was a little tricky getting the logrotate files where I wanted them when I was making changes on them in November